backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Controllers/DelegationsController.cs

```diff
@@ -213,6 +217,29 @@ public async Task<ActionResult> DeletePolicy([FromBody] RequestToDeletePolicyLis
             return StatusCode(400, $"Unable to complete deletion");
         }
 
+        /// <summary>
+        /// Endpoint for retrieving delegated resources between parties
```

Comment:
Summary describes a generic resource delegations but implementation and operation name is API delegation specific.

Response model is generic named DelegatedResources, is model structured specifically for the API delegation views and should be named specifically for API delegation?

Reply:
I have made it geenric now. Response model is also not specific to api. I have changed the name to OfferedDelegation.